pt/lessons/ecto/basics.md

```diff
@@ -1,360 +1,243 @@
 ---
-version: 1.3.0
-title: Basics
+version: 2.4.0
+title: Básico
 ---
 
-Ecto é um projeto oficial do Elixir que fornece uma camada de banco de dados e linguagem integrada para consultas. Com Ecto podemos criar *migrations*, definir schemas, inserir e atualizar registos, e fazer consultas.
+Ecto é um projeto oficial do Elixir que fornece uma camada de banco de dados e linguagem integrada para consultas. Com Ecto podemos criar migrações, definir esquemas, inserir e atualizar registros, e fazer consultas.
 
 {% include toc.html %}
 
-## Instalação
+### Adaptadores
+O Ecto suporta diferentes banco de dados através do uso de adaptadores. Alguns
+exemplos de adaptadores são:
 
-Crie um novo app com uma supervision tree:
++ PostgreSQL
++ MySQL
++ SQLite
 
-```shell
-$ mix new example_app --sup
-$ cd example_app
-```
+Nessa lição configuraremos o Ecto para usar o adaptador do PostgreSQL.
 
-Para começar precisamos incluir Ecto e um adaptador de banco de dados no `mix.exs` do nosso projeto. Você pode encontrar uma lista de adaptadores de banco de dados suportados na seção [*Usage*](https://github.com/elixir-lang/ecto/blob/master/README.md#usage) do README do Ecto. Para o nosso exemplo iremos usar o PostgreSQL:
+### Começando
+Nesta lição, cobriremos três partes do Ecto:
 
-```elixir
-defp deps do
-  [{:ecto, "~> 2.2"}, {:postgrex, ">= 0.0.0"}]
-end
-```
++ O repositório: provê a interface com nosso banco de dados, incluindo a conexão.
++ Migrações: um mecanismo para criar, modificar e destruir tabelas e índices no
+  banco de dados.
++ Esquemas: estruturas especiais para representar linhas em tabelas no banco de
+  dados.
 
-Então nós vamos baixar nossas dependências usando
+Para iniciar criaremos uma aplicação com uma árvore de supervisão:
 
 ```shell
-$ mix deps.get
+$ mix new friends --sup
+$ cd friends
 ```
 
-### Repositório
-
-Finalmente precisamos criar o repositório do nosso projeto, a camada de banco de dados. Isto pode ser feito rodando a tarefa `mix ecto.gen.repo -r ExampleApp.Repo`, falaremos sobre tarefas mix no Ecto mais para frente. O Repositório pode ser encontrado no arquivo `lib/<nome_do_projecto>/repo.ex`:
+Adicione o ecto e o postgrex como dependências no seu `mix.exs`:
 
 ```elixir
-defmodule ExampleApp.Repo do
-  use Ecto.Repo, otp_app: :example_app
+defp deps do
+  [
+    {:ecto_sql, "~> 3.2"},
+    {:postgrex, "~> 0.15"}
+  ]
 end
 ```
 
```

Comment:
Looking at the English version, there is an instruction to fetch dependencies here. Suggestion:

Busque as dependências usando

```shell
$ mix deps.get
```